plugins/cp/cp.plugin.zsh

```diff
@@ -1,4 +1,4 @@
 cpv() {
-    rsync -pogbr -hhh --backup-dir=/tmp/rsync -e /dev/null --progress "$@"
+    rsync -pogbr -hhh --backup-dir="/tmp/rsync-${USER:-"$(whoami)"}" -e /dev/null --progress "$@"
```

Comment:
You can use `$USERNAME` which [is always guaranteed to be defined inside a Zsh session](https://zsh.sourceforge.io/Doc/Release/Parameters.html#index-USERNAME).